diff --git a/pysyncthru/tests/test_web.py b/pysyncthru/tests/test_web.py
index b701f9b..d0f2e1a 100644
--- a/pysyncthru/tests/test_web.py
+++ b/pysyncthru/tests/test_web.py
@@ -58,7 +58,7 @@ async def fetch():
                 )
                 await self.syncthru.update()
 
-        loop = asyncio.get_event_loop()
+        loop = asyncio.new_event_loop()
         loop.run_until_complete(fetch())
 
     def test_online(self) -> None:
@@ -254,7 +254,7 @@ async def fetch():
                 )
                 await self.syncthru.update()
 
-        loop = asyncio.get_event_loop()
+        loop = asyncio.new_event_loop()
         loop.run_until_complete(fetch())
 
     def test_newline_unescaped_status(self) -> None:
@@ -305,7 +305,7 @@ async def fetch():
                 )
                 await self.syncthru.update()
 
-        loop = asyncio.get_event_loop()
+        loop = asyncio.new_event_loop()
         loop.run_until_complete(fetch())
 
     def test_location(self) -> None:
@@ -382,7 +382,7 @@ async def fetch() -> None:
                     )
                     await self.syncthru.update()
 
-            loop = asyncio.get_event_loop()
+            loop = asyncio.new_event_loop()
             loop.run_until_complete(fetch())
             self.fail(
                 "No error thrown when noticing that the host does not support Syncthru"
@@ -400,7 +400,7 @@ async def fetch() -> None:
                 )
                 await self.syncthru.update()
 
-        loop = asyncio.get_event_loop()
+        loop = asyncio.new_event_loop()
         loop.run_until_complete(fetch())
         self.assertFalse(self.syncthru.is_online())
         self.assertTrue(self.syncthru.is_unknown_state())
